statsd-exporter/include/mappings-gen2.yml

```diff
@@ -101,6 +101,14 @@ mappings:
     name: "airflow_executor_queued_tasks"
     labels: {}
 
+  - match: airflow.executor.*.resources.*
```

Comment:
I think that this line is matching [this metric](https://github.com/astronomer/stellar/blob/59e49dc5030ebc0e50240a9a1e34edde7ee44379/stellar/allocator/allocator.py#L493) ?

If so I don't think it would be prefixed with `airflow.` ? Wouldn't this be: `- match: executor.*.resources.*`

Reply:
I have a corresponding PR on stellar to update these metrics and ensure they are unique to the mapping.yml. https://github.com/astronomer/stellar/pull/303

Reply:
I have a corresponding PR to update these names on the executor.